src/quo2/screens/community/discover_card.cljs

```diff
@@ -0,0 +1,36 @@
+(ns quo2.screens.community.discover-card
+  (:require [quo.react-native :as rn]
+            [quo.previews.preview :as preview]
+            [reagent.core :as reagent]
+            [status-im.i18n.i18n :as i18n]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.community.discover-card :as discover-card]))
+
+(def descriptor [{:label   "Discover card"
+                  :type    :select}
```

Comment:
do we need DIscover card selector? there's no options here 🤔

Reply:
right we dont need that